src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -73,51 +76,60 @@ export const RegistrationFormActions = ({
-              {tabNumber === RegistrationTab.FilesAndLicenses && t('registration.heading.contributors')}
-            </Button>
-          </Box>
-        )}
+        <Box sx={{ gridArea: 'back-button' }}>
+          <IconButton size="large" onClick={() => setTabNumber(tabNumber - 1)} disabled={isFirstTab}>
+            <Box
+              sx={{
+                bgcolor: isFirstTab ? 'gray' : 'info.main',
+                display: 'flex',
+                borderRadius: '50%',
+                padding: '0.2rem',
+              }}>
+              <ChevronLeftIcon sx={{ color: 'white' }} />
+            </Box>
+          </IconButton>
+        </Box>
+
         <Button data-testid="open-support-button" onClick={toggleSupportModal} sx={{ gridArea: 'support-button' }}>
           {t('common.support')}
         </Button>
         {tabNumber < RegistrationTab.FilesAndLicenses ? (
-          <Box
-            sx={{
-              gridArea: 'save-next-button',
-              display: 'grid',
-              gridTemplateAreas: '"save-button next-button"',
-              columnGap: '1rem',
-            }}>
-            <LoadingButton
-              variant="outlined"
-              loading={isSaving}
-              data-testid="button-save-registration"
-              onClick={async () => {
-                await saveRegistration(values);
-                // Set all fields with error to touched to ensure error messages are shown
-                setTouched(setNestedObjectValues(errors, true));
+          <>
+            <Box
+              sx={{
+                gridArea: 'save-next-button',
+                display: 'grid',
+                gridTemplateAreas: '"save-button next-button next-tab-button"',
+                columnGap: '1rem',
               }}>
-              {t('common.save')}
-            </LoadingButton>
-            <Button
-              variant="contained"
-              data-testid="button-next-tab"
-              endIcon={<ArrowForwardIcon />}
-              onClick={() => setTabNumber(tabNumber + 1)}>
-              {tabNumber === RegistrationTab.Description && t('registration.heading.resource_type')}
-              {tabNumber === RegistrationTab.ResourceType && t('registration.heading.contributors')}
-              {tabNumber === RegistrationTab.Contributors && t('registration.heading.files_and_license')}
-            </Button>
-          </Box>
+              <LoadingButton
+                variant="outlined"
+                loading={isSaving}
+                data-testid="button-save-registration"
+                onClick={async () => {
+                  await saveRegistration(values);
+                  // Set all fields with error to touched to ensure error messages are shown
+                  setTouched(setNestedObjectValues(errors, true));
+                }}>
+                {t('common.save')}
+              </LoadingButton>
+              <IconButton
```

Comment:
Skulle hatt title eller tooltip på denne. Usikker på om det skal være "Neste" eller navnet på neste panel som det var før... 🤔 Lener litt mot sistnevnte.
Samme gjelder for forrige-knappen

Reply:
"Neste, bidragsytere" etc er også en mulighet